tst/testinstall/perm.tst

```diff
@@ -208,7 +208,7 @@ gap> List(permAll, x -> x^-3);
 # PowIntPerm
 #
 gap> 0^(1,2);
-Error, Perm. Operations: <point> must be a positive integer (not 0)
+Error, PermImage: <point> must be a positive small integer (not the integer 0)
 gap> n:=10^30;;
 gap> ForAll(permAll, g -> n^g = n);
```

Comment:
Note the contradiction between this test, and the error message above (10^30 is not a positive small integer).